.github/workflows/Tests.yml

```diff
@@ -53,6 +53,18 @@ jobs:
 
     steps:
 
+      - name: Installing Klayout
+        run: |
+          sudo apt update -qq -y
+          wget https://www.klayout.org/downloads/Ubuntu-22/klayout_0.28.2-1_amd64.deb
+          # The dpkg command will fail complaining about missing dependencies.
+          sudo dpkg -i ./klayout_0.28.2-1_amd64.deb || true
+          # The apt install command should install the missing dependencies
+          # needed by KLayout above and finish the install.
+          sudo apt install -f -y
+          # Check that KLayout was successfully installed!
+          klayout --version
```

Comment:
This is what `klayout --version` returns:
```
(base) ~ 🍙 klayout --version
ERROR: Unknown option: --version
```
I suspect it's:
```
klayout -v
```

Reply:
Tried `klayout --help` instead?